doc/build-unix.md

```diff
@@ -326,7 +326,7 @@ This example lists the steps necessary to setup and build a command line only, n
 Note:
 Enabling wallet support requires either compiling against a Berkeley DB newer than 4.8 (package `db`) using `--with-incompatible-bdb`,
 or building and depending on a local version of Berkeley DB 4.8. The readily available Arch Linux packages are currently built using
-`--with-incompatible-bdb` according to the [PKGBUILD](https://projects.archlinux.org/svntogit/community.git/tree/bitcoin/trunk/PKGBUILD).
+`--with-incompatible-bdb` according to the [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=bitcoin-git).
```

Comment:
Concept ACK on fixing the link, which appears to be broken.

If the new link is the right one and I'm reading it correctly, line 12 seems to indicate that db4.8 is a dependency rather than `--with-incompatible-bdb`.

Reply:
This is wrong. The new link points to a PKGBUILD to the (semi-offical) Arch User Repository. The official PKGBUILD is here: https://github.com/archlinux/svntogit-community/blob/packages/bitcoin/trunk/PKGBUILD, see the "Source Files" link here: https://archlinux.org/packages/community/x86_64/bitcoin-daemon/